lib/postgresql.js

```diff
@@ -15,6 +15,11 @@ var ParameterizedSQL = SqlConnector.ParameterizedSQL;
 var util = require('util');
 var debug = require('debug')('loopback:connector:postgresql');
 
+// Add support for node 0.10.x
+if (typeof Promise == 'undefined') {
+  global.Promise = require('promise-polyfill');
```

Comment:
> cis-jenkins builds are now working - and failing on node 0.10 due to Promise not being defined.

I was looking into it and `pg` has `pg-pool` which states this:
https://www.npmjs.com/package/pg-pool#bring-your-own-promise